app/forms/idv/doc_pii_form.rb

```diff
@@ -4,7 +4,7 @@ class DocPiiForm
 
     validate :validate_pii
 
-    attr_reader :first_name, :last_name, :dob, :state, :zipcode, :attention_with_barcode,
+    attr_reader :first_name, :last_name, :dob, :address1, :state, :zipcode, :attention_with_barcode,
```

Comment:
>  If either the front/back has a missing address we'd expect a failure, but if both are has-no-errors.yml we expect the document upload to succeed.

**TLDR**: I did not see this behavior when I tested manually, but I think the issue is with the mock client, not with your code changes.


I saw some unexpected behavior when I tested with a mix of images.
1. I see an address error when the front image has no errors and the back image is missing address1.  
2. I see no errors when the front image is missing address1 and the back image has no errors.


Here’s `client_response.pii_from_doc` in the first scenario
```
=> {:type=>"license",
:first_name=>"Susan",
:last_name=>"Smith",
:middle_name=>"Q",
:address1=>"",
:address2=>"Apt 3",
:city=>"Bayside",
:state=>"NY",
:zipcode=>"11225",
:dob=>"1938-10-06",
:phone=>"+1 314-555-1212",
:state_id_number=>"123456789",
:state_id_type=>"drivers_license",
:state_id_jurisdiction=>"NY"}
```



Here’s  `client_response.pii_from_doc` in the second scenario

```
=> {:type=>"license",
:first_name=>"Susan",
:last_name=>"Smith",
:middle_name=>"Q",
:address1=>"1 Microsoft Way",
:address2=>"Apt 3",
:city=>"Bayside",
:state=>"NY",
:zipcode=>"11364",
:dob=>"1938-10-06",
:phone=>"+1 314-555-1212",
:state_id_number=>"123456789",
:state_id_type=>"drivers_license",
:state_id_jurisdiction=>"NY"}
```

I poked around the mock client and `result_response.rb`, then found the method `pii_from_doc` [in the later file](https://github.com/18F/identity-idp/blob/main/app/services/doc_auth/mock/result_response.rb#L57-L64).  The mock client creates the result response [with the last posted back image](https://github.com/18F/identity-idp/blob/main/app/services/doc_auth/mock/doc_auth_mock_client.rb#L82).  This explains the unexpected behavior.

This unexpected behavior highlights an opportunity for refactoring the mock client.  However, that is out of scope for this PR.